utils/das_data_sorter.py

```diff
@@ -72,5 +148,15 @@ def avg_Data(array):
     "power": avg_Data(power),
     "reed_velocity": avg_Data(reed_vel),
     "reed_distance": avg_Data(reed_dis)
-})
+}
+
+# checks if user wants to smooth data using moving mean or moving median method
+# if no input from user, script will ignore this whole block
+if args.smooth == "mean":
```

Comment:
Seems like a unnecessary if statement here? We can simply just pass in `args.smooth`